configs/gcp.json

```diff
@@ -5,7 +5,7 @@
     "spotCPU": "0.006655",
     "RAM": "0.004237",
     "spotRAM": "0.000892",
-    "projectID": "guestbook-227502",
+    "projectID": "my-awesome-project",
```

Comment:
Is there a valid string we can provide that has invalid GCP project ID characters to guarantee no overlap with a valid ID? The same question goes for all fields; having something that is too short to be valid also counts, I think!

Reply:
So GCP project IDs must be 6-30 characters in length [source](https://cloud.google.com/resource-manager/docs/creating-managing-projects). AWS S3 bucket names must be between 3 and 63 characters in length [source](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-s3-bucket-naming-requirements.html). And unfortunately AWS Athena table names can be anything less than 255 characters [source](https://docs.aws.amazon.com/athena/latest/ug/tables-databases-columns-names.html). Values chosen for each accordingly.